set.current(processors=10)
make.file(type=fastq, prefix=COI)
make.contigs(file=current, insert=30)
summary.seqs(fasta=current)
screen.seqs(fasta=current, group=current, summary=current, minlength=200, maxambig=0)
summary.seqs(fasta=current)
unique.seqs(fasta=current)
summary.seqs(fasta=current, name=current)
count.seqs(name=current, group=current)
#using MetaZooGene Global database as reference
align.seqs(fasta=current, reference=MZGGlobalCOI7April2020.MAFFT.fasta, flip=t)
summary.seqs(fasta=current, count=current)
screen.seqs(fasta=current, count=current, summary=current, minlength=200, maxhomop=8)
summary.seqs(fasta=current, count=current)
unique.seqs(fasta=current, count=current)
summary.seqs(fasta=current, count=current)
classify.seqs(fasta=current, count=current, reference=ArCOP.COI.Mothur.ngV2.fasta, taxonomy=TaxMap.ArCOP.COI.MothurV2.txt, cutoff=90)
#Need version 1.44 or newer
make.shared(count=current, label=asv)
classify.otu(list=current, count=current, taxonomy=current, label=asv)